### ISSUE TITLE
Add switches?

### ISSUE THREAD

**Issue by gmeader** [code2k13/pipico_desk_display#1]
This display board is wonderful. 
It would be so great to have an option on the PCB to add 3 or 4 tactile pushbuttons next to the display, so that it would provide a complete UI on one board.  Like this: https://www.adafruit.com/product/5691
Or even a rotary encoder?
If buttons were not needed or would be located elsewhere, the parts could simply be omitted when building it.

**Comment by code2k13** [comment]
Thats a very good suggestion. Where would you prefer the buttons to be, on the back side ? I am working on a case for the display that only covers the front section. The Pi Pico is not covered by the case. That would allow adding stackable headers and more components to the pi. So I feel the back side would be a good place to add buttons.

On the Adafruit board they are not using headers to connect the LCD, so the buttons are on the same level. Maybe if we omit headers while connecting the LCD , buttons on front make more sense. I will give it a go over the weekend. It might be possible to allow connecting buttons on either side, I need to give it a try. 

My only trouble is I don't have easy access to PCB fab. They will at least make 5 pieces and take a month :-) to deliver. But I will create a branch and start work on v2 on the board.

**Comment by gmeader** [comment]
Yes, I agree that buttons on the front are the only way that makes sense. A
user is looking at the LCD and pressing buttons related to what they are
seeing on the display.

Tactile switches do not need to be at the same level as the LCD, as you can
use 3D printed buttons to extend the height of the surface that a finger
pushes. These can extend through holes in the case.

Also consider using an encoder with push switch instead of, or in addition
to tactile pushbutton switches. An encoder can make a very simple and
elegant UI.  https://www.youtube.com/watch?v=mVNaiqcpl4w

https://exploreembedded.com/wiki/Interactive_Menus_for_your_project_with_a_Display_and_an_Encoder

Maybe you can find a vendor that can deliver PCB much faster than a month.
I found a bunch of them by Googling: fast pcb

Intersting project!

**Comment by code2k13** [comment]
How about using a 5D Joystick , like this

https://robu.in/product/5d-rocker-joystick-for-arduino-mcu-avr-game/?gclid=CjwKCAjwzuqgBhAcEiwAdj5dRsOMbKcaXXWqZRQI6yzYw9tbc61KpOHK0aY4CRp0kI1Lt8FAUKjR_RoC5esQAvD_BwE

Using an angled header, it might be easier to add.

This one comes without breakout board. Should save some space on PCB

https://robu.in/product/thumb-joystick-button/?gclid=CjwKCAjwzuqgBhAcEiwAdj5dRqtK8r7oRp4IQxkugpZDaAMNnBdq5-g8u48qUJfX2uA59ktT-ZgTdBoCJrIQAvD_BwE
